<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>YouTubeClipGeneratorClass</class>
 <widget class="QMainWindow" name="YouTubeClipGeneratorClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>421</width>
    <height>122</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>YouTubeClipGenerator</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="clip_button">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>90</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>clip</string>
    </property>
    <property name="shortcut">
     <string>C</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>10</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>To</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="name_input">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>40</y>
      <width>281</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Name</string>
    </property>
   </widget>
   <widget class="QLabel" name="status_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>281</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="file_name_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>281</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Time</string>
    </property>
   </widget>
   <widget class="QPushButton" name="open_folder">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>90</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Open folder</string>
    </property>
   </widget>
   <widget class="QTimeEdit" name="start">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>10</y>
      <width>118</width>
      <height>22</height>
     </rect>
    </property>
    <property name="displayFormat">
     <string>mm:ss</string>
    </property>
   </widget>
   <widget class="QTimeEdit" name="end">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>10</y>
      <width>118</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximumTime">
     <time>
      <hour>4</hour>
      <minute>59</minute>
      <second>59</second>
     </time>
    </property>
    <property name="currentSection">
     <enum>QDateTimeEdit::MinuteSection</enum>
    </property>
    <property name="displayFormat">
     <string>mm:ss</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="YouTubeClipGenerator.qrc"/>
 </resources>
 <connections/>
</ui>
